packages/facebook-oauth/package.js

```diff
@@ -0,0 +1,18 @@
+Package.describe({
+  summary: "Facebook OAuth flow",
+  version: "1.3.0"
+});
+
+Package.onUse(function(api) {
+  api.use('oauth2', ['client', 'server']);
+  api.use('oauth', ['client', 'server']);
+  api.use('http', ['server']);
+  api.use('underscore', 'server');
+  api.use('random', 'client');
+  api.use('service-configuration', 'client');
```

Comment:
My fault. I did test the deprecation notice and roughly use a code-configured service to test the ui.